src/code/z_play.c

```diff
@@ -905,10 +912,10 @@ void Gameplay_Update(GlobalContext* globalCtx) {
             if (globalCtx->unk_1242B != 0) {
                 if (CHECK_BTN_ALL(input[0].press.button, BTN_CUP)) {
                     if ((globalCtx->pauseCtx.state != 0) || (globalCtx->pauseCtx.debugState != 0)) {
-                        // Translates to: "Changing viewpoint is prohibited due to the kaleidoscope"
+                        // "changing viewpoint is prohibited due to being in the kaleidoscope"
                         osSyncPrintf(VT_FGCOL(CYAN) "カレイドスコープ中につき視点変更を禁止しております\n" VT_RST);
                     } else if (Player_InCsMode(globalCtx)) {
-                        // Translates to: "Changing viewpoint is prohibited during the cutscene"
+                        // "changing viewpoint is prohibited due to being in a demo"
```

Comment:
We use "cutscene" instead of "demo" wherever that is what is implied, for example here the line right above has `Player_InCsMode` where `Cs` stands for "Cutscene"

I'm not sure if we also want to apply this to translations, more opinions needed